doc/man3/RSA_generate_key.pod

```diff
@@ -2,32 +2,37 @@
 
 =head1 NAME
 
+OSSL_RSA_keygen,
```

Comment:
I am not sure this belongs in this file..
It should probably be in L<EVP_PKEY-RSA(7)> and just be pointing to there from this file..

Reply:
`L<EVP_PKEY-RSA(7)>` does not document actual function names,
and people would expect RSA key gen functions described in `RSA_generate_key.pod`.
Moreover, it make sense to describe the new function there as a simple replacement for the most common use of the deprecated functions. 

BTW, the function meanwhile is called `EVP_RSA_keygen`.

Good point though to add an example using the new simple function to `L<EVP_PKEY-RSA(7)>`:

    EVP_RSA_keygen(NULL, 4096, NULL);

Reply:
Likewise added
```
    EVP_EC_keygen(NULL, "P-256", NULL);
```
to `EVP_PKEY-EC.pod`.

Reply:
> expect RSA key gen functions described in `RSA_generate_key.pod`.
> Moreover, it make sense to describe the new function there as a simple replacement for the most common use of the deprecated functions.

Maybe EVP_PKEY_gen.pod is a better place for the functions with a linl in RSA_generate_key.pod?

Reply:
Would be an option.
Currently  EVP_PKEY_gen.pod contains the doc of `EVP_keygen()`
and contains
```
 =head1 SEE ALSO

L<EVP_RSA_keygen(3)>, L<EVP_EC_keygen(3)>,
```
which I find slightly more adequate.

Reply:
This would be less of a problem with non pod-like documentation that had more logical structure :)

Reply:
@slontis, Unix man page section 3 is for library functions.  Documenting the functions from this PR in section 7 is simply wrong in that sectioning.

That being said, if one can't find a suitable file, *create a new file*!